pkg/projects/dir.props

```diff
@@ -57,19 +57,11 @@
 
   <ItemGroup>
     <OfficialBuildRID Include="alpine.3.4.3-x64" />
-    <OfficialBuildRID Include="debian.8-x64" />
     <OfficialBuildRid Include="debian.8-armel">
       <Platform>armel</Platform>
     </OfficialBuildRid>
-    <OfficialBuildRID Include="fedora.23-x64" />
```

Comment:
No.  The RID that the machine is building for gets added to this list implicitly.  See https://github.com/dotnet/core-setup/blob/master/pkg/projects/dir.props#L87-L91

The latest fedora 23 build passed:  https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_build?_a=summary&buildId=634406